tests/integ/modin/series/test_getattr.py

```diff
@@ -16,7 +16,8 @@
 @pytest.mark.parametrize(
     "name, expected_query_count, expected_join_count",
     [
-        ("a", 2, 2),
+        # Upstream Modin performs a check against self.index["a"] rather than self["a"], incurring an additional query.
+        ("a", 3, 2),
```

Comment:
that is counted as regression, can we overwrite the corresponding implementation